content/dart/concepts/map/map.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Map'
+Description: 'The Map object is a simple key/value pair.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Map'
+  - 'Dart'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+## Syntax
+
+The syntax to declare a `map` is -
```

Comment:
```suggestion
The syntax to declare a `map` is:
```

Reply:
The changes have been made. Please let me know anything else I can improve. Thank you!